examples/stage/odc_k8s/odc_k8s.tf

```diff
@@ -10,29 +10,42 @@ data "terraform_remote_state" "odc_eks-stage" {
 data "aws_caller_identity" "current" {
 }
 
+# collecting data from odc_eks module
+locals {
```

Comment:
Is this for readability? Not sure I prefer the additional indirection required to figure out where the var is coming from

Reply:
Mainly for readability.